schemas/session-replay/mobile/_common-shape-wireframe-schema.json

```diff
@@ -16,7 +16,8 @@
           "properties": {
             "backgroundColor": {
               "type": "string",
-              "description": "The background color for this wireframe as a String hexadecimal. Follows the #RRGGBBAA color format, default value should be #FFFFFF00.",
+              "pattern": "^[#][a-f0-9]{6}([a-f0-9]{2})?$",
```

Comment:
**question:** why keep the bracket around the `#` It's not necessary, right?

Reply:
Well....is the way you define a color as a hexa string. I would rather keep it.

Reply:
Yeah but in the regex, you can change `^[#][a-f0-9]{6}([a-f0-9]{2})?$` to `^#[a-f0-9]{6}([a-f0-9]{2})?$`